src/main/kotlin/ink/abb/pogo/scraper/Main.kt

```diff
@@ -118,6 +118,23 @@ fun main(args: Array<String>) {
         println("No item drop policy specified, defaulting to $shouldDropItems")
     }
 
+    if(shouldDropItems){
+        HashMap(uselessItems).forEach {
+            val itemName = it.key.name.toLowerCase()
+
+            try {
+                val newValue = properties.getProperty(itemName).toInt()
```

Comment:
It will use the defaults already in uselessItems.

The val uselessItems is an mutableMapOf so you can update the map. The map is in 332 and already has the default values.

If some properties aren't in the config file or aren't valid it will inform about the default used.